test/server/reporters-test.js

```diff
@@ -336,4 +352,12 @@ describe('Reporters', function () {
     it('xunit', function (done) {
         testReporter('xunit', done);
     });
+
+    it('Should xunit reporter use plain text decorator', function () {
+        testUsingPlainTextDecorator('xunit');
+    });
+
+    it('Should json reporter use plain text decorator', function () {
```

Comment:
Even if it not readable 
```
Should reporters that target machine-readable formats always use the plain text decorator
```
still doesn't makes much sense, since it has question word order =)